python/tests/test_common.py

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# Copyright (c) 2020 Scipp contributors (https://github.com/scipp)
+# @author Simon Heybrock
+import pytest
+import scipp as sc
+
+
+def test_bool_raises():
+    # Truth values of arrays are undefined
+    var = sc.Variable(value=True)
+    with pytest.raises(RuntimeError):
+        var and var
+    da = sc.DataArray(var)
+    with pytest.raises(RuntimeError):
+        da and da
+    ds = sc.Dataset({'a': da})
+    with pytest.raises(RuntimeError):
+        ds and ds
```

Comment:
Shouldn't this behaviour AND the fixed behaviour of `==` vs. `is_equal` for Variables be described/fixed in documentation as well?

Reply:
Hmm, good point, I thought `__eq__` etc. should be listed in https://scipp.github.io/generated/scipp.Variable.html#scipp.Variable, but for some reason it does not show up. Will add updating docstringsto the TODO list, since it is unrelated to this change.

`is_equal` is listed here: https://scipp.github.io/python-reference/api.html#comparison.